cli/azd/internal/appdetect/spring_boot.go

```diff
@@ -93,6 +93,7 @@ func detectAzureDependenciesByAnalyzingSpringBootProject(
 		parentProject:         parentProject,
 		mavenProject:          mavenProject,
 	}
+	detectSpringApplicationName(azdProject, &springBootProject)
```

Comment:
Could you please create another PR to merge `detectSpringApplicationName` , `detectSpringCloudAzure`, `detectSpringCloudEureka`, `detectSpringCloudConfig` into new func: `detectMetadata`?

Reply:
Updated in this PR https://github.com/azure-javaee/azure-dev/pull/63